### PR TITLE
chore(release): update changelog for v0.27.0 release 

### PR DESCRIPTION
**Pull request by gabrielmer**
# Description

Changelog for v0.27.0

Please review carefully and suggest highlights, upgrade instructions or notes that should be added :)

# Changes

- [x] updated changelog

### CODE REVIEW

**Review by jm-clius — APPROVED** [review]
LGTM. Thanks! :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,80 @@
+## v0.27.0 (2024-04-19)
+
+>  **Note:**
+
+>  - Filter v1 protocol and its REST-API access have been deprecated.
+>  - A new field of the `WakuMetadataRequest` protobuf for shards was introduced. The old shards field (2) will be deprecated in 2 releases time
+> - CLI flags `--requestRateLimit` and `requestRatePeriod` have been added for rate limiting configuration.
```
Comment (NagyZoltanPeter) [inline CHANGELOG.md:7]:
```suggestion
>  - CLI flags `--requestRateLimit` and `--requestRatePeriod` have been added for rate limiting configuration. Period is measured in seconds. Limits are measured per protocol per period of time. Over limit will result in TOO_MANY_REQUEST (429) response.
```

**Review by NagyZoltanPeter — APPROVED** [review]
LGTM, thank you for it!

File: CHANGELOG.md (hunk 1 above)
Comment (gabrielmer) [reply CHANGELOG.md:7]: Ooh amazing! Thanks for the improvement! 😍

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks! :100:

**Pull request merged**